packages/template/vite-typescript/tmpl/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "ESNext",
+    "module": "ESNext",
```

Comment:
Should this be `commonjs`?

See my messages in main PR discussion here: https://github.com/electron/forge/pull/3178#issuecomment-1592454485 and here: https://github.com/electron/forge/pull/3178#issuecomment-1606555412

Reply:
Okay! 👍

Reply:
@MarshallOfSound can you confirm this is the correct guidance? I know I have seen you discuss this in other areas like here: https://github.com/electron/forge/issues/3129#issuecomment-1370798467 and want to make sure this is still the correct guidance / path forward for the forge-vite-typescript template?